backend/services/create_battle.py

```diff
@@ -0,0 +1,36 @@
+from users.models import User
+from django.utils.crypto import get_random_string
+from django.contrib.auth.forms import PasswordResetForm
+from battle.battles.email import send_invite_email
+from django.conf import settings
+
+
+def validate_if_creator_and_opponent_are_different(creator, opponent):
```

Comment:
A name suggestion here is: `has_different_contenders`

Reply:
We always try to name functions that return booleans with `has` and `is`.

Reply:
great! Good to know it :)

Reply:
This was not updated :eyes:

Reply:
![image](https://user-images.githubusercontent.com/53096507/129402404-0486cb48-a2ac-4472-848f-0d22f6267b80.png)
You said here to change it back haha

Reply:
Alright, I'm going to change to `validate_id_creator_and_opponent_has_different_contenders,` right?

Reply:
:eyes: bad advice, `has_different_contenders` is better